pyproject.toml

```diff
@@ -79,7 +78,7 @@ tests = [
 ]
 rtd = ["sphinx>=5.1.0,<6", "pygments", "pydot", "pydot2", "pydot-ng"]
 jax = ["jax", "jaxlib"]
-numba = ["numba>=0.57"]
+numba = ["numba>=0.57", "llvmlite"]
```

Comment:
Isn't llvmlite a dependency of numba? Does pytensor call llvmlite?

Reply:
Yes and yes. This is nothing more than a guard against the possibility of numba dropping the llvmlite dependency and causing breakage on our side. While it's pretty obscure, it was flagged by #679, and I think it's a good idea.

Reply:
Indeed: https://github.com/pymc-devs/pytensor/blob/193580990e9f7e7eebca98af217b960929656d29/pytensor/link/numba/dispatch/basic.py#L14